native/rust/archetype/pom.xml

```diff
@@ -77,8 +77,7 @@
               <goal>integration-test</goal>
             </goals>
             <configuration>
-              <!-- adding -DskipITs can be used to skip this IT from the command line -->
-              <skip>false</skip>
+              <skip>${skipTests}</skip>
```

Comment:
i don't see this property defined anywhere; where is it defined?

Reply:
It's the same property used by the Surefire plugin to skip unit tests https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#skipTests

I reused this flag to skip this integration test as well, since the `skipITs` flag is less well-known.

Reply:
let's leave a comment about this info too then since it is not obvious unless you know the details about the surefire test plugin